samples/boards/arc_secure_services/nsim_sem_normal_defconfig

```diff
@@ -7,6 +9,7 @@ CONFIG_BUILD_OUTPUT_BIN=n
 CONFIG_PRINTK=y
 CONFIG_ARCV2_INTERRUPT_UNIT=y
 CONFIG_ARCV2_TIMER=y
+CONFIG_ARC_MPU_ENABLE=y
```

Comment:
Is the MPU now supported in the 'normal' image?

Reply:
Yes, but maybe I should undo this modification until future pull request to actually support MPU?

Reply:
Yes, I think it is better to focus this PR on support for no-multithreading only.